src/core_plugins/kibana/public/management/sections/indices/edit_index_pattern/scripted_field_editor/scripted_field_editor.js

```diff
@@ -43,7 +43,11 @@ uiRoutes
         this.field = this.indexPattern.fields.byName[fieldName];
 
         if (!this.field) {
-          notify.error(this.indexPattern + ' does not have a "' + fieldName + '" field.');
+          toastNotifications.add({
+            title: `"${this.indexPattern.title}" index pattern doesn't have a scripted field called "${fieldName}"`,
```

Comment:
Do you want to use single quotes to be consistent with the confirm modals?

Reply:
Yes! Great catch. Will fix.